llvm/lib/Target/AMDGPU/AMDGPU.td

```diff
@@ -303,6 +303,12 @@ def FeatureMSAALoadDstSelBug : SubtargetFeature<"msaa-load-dst-sel-bug",
   "MSAA loads not honoring dst_sel bug"
 >;
 
+def FeaturePrivEnabledBug : SubtargetFeature<"priv-enabled-bug",
```

Comment:
Should this mention trap, or is it more general?

Reply:
The reason we need a simulated trap is because priv=1 is enabled, but AFAIK simulated trap is the only effect of that we care about. Would you prefer naming this FeatureTrap2NopBug or something like that?

Reply:
Well I think it should be more descriptive than "priv enabled bug" in case there are different priv enabled bugs in the future. Doesn't necessarily mean it involves trap

Reply:
I renamed it `FeaturePrivEnabledTrap2NopBug` in the update. WDYT? 

Incidentally, I noticed that FeatureMADIntraFwdBug isn't included in the gfx11-generic FeatureSet. Should that be updated too?

Reply:
> Incidentally, I noticed that FeatureMADIntraFwdBug isn't included in the gfx11-generic FeatureSet. Should that be updated too?

I guess so, but not in this patch. @Pierre-vh?

Reply:
Ok thanks, I just created #89936 to fix this.